src/System.Collections/tests/Generic/Dictionary/Dictionary.Generic.Tests.netcoreapp.cs

```diff
@@ -84,5 +84,52 @@ public void Dictionary_Generic_RemoveKey_DefaultKeyContainedInDictionary(int cou
         }
 
         #endregion
+
+        #region EnsureCapacity
+        
+        [Fact]
+        public void EnsureCapacity_NegativeCapacityRequested_Throws()
+        {
+            Dictionary<string, string> dictionary = new Dictionary<string, string>();
```

Comment:
Nit, where the right hand side is a simple `new` you can use var for the left side as it's obvious what the type is.

Reply:
it's less wordy